cypress/e2e/tests/administration/questionnaires/miscellaneous.test.ts

```diff
@@ -64,6 +64,17 @@ describe(["@tier3"], "Miscellaneous Questinnaire tests", () => {
         click(closeModal);
     });
 
+    it("Performs question search on questionnaires", function () {
+        // Automates Polarion - MTA 435
+        AssessmentQuestionnaire.open();
+        AssessmentQuestionnaire.view(legacyPathfinder);
+        const textInput = "Production";
+        AssessmentQuestionnaire.searchQuestions(textInput);
+        AssessmentQuestionnaire.validateNumberOfMatches("Application details", 4);
+        AssessmentQuestionnaire.validateNumberOfMatches("Application cross-cutting concerns", 1);
+        AssessmentQuestionnaire.validateSearchWordInRows(textInput);
```

Comment:
The last step of TC is yet to be covered .

Clear search box; Enter random string in the search box	After step 3 , the following message should get displayed: No questions match your search

And one more TC , Click on " Back to questionnaire" should navigate to main page .

![Screenshot from 2024-01-18 12-01-43](https://github.com/konveyor/tackle-ui-tests/assets/3866976/9efca113-c54f-488b-a27e-f4a57e0c97e7)

Reply:
Done and updated jenkins